doc/installation.md

```diff
@@ -27,10 +27,9 @@ You will need:
 - Python 3
 - [Libvips](https://libvips.github.io/libvips/install.html)
 - pyvips (install it via python pip: `pip install pyvips`) 
-- [compose.py](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/tools/gfx_tools/compose.py) script from the main [Cataclysm-DDA](https://github.com/CleverRaven/Cataclysm-DDA) repository
 
 Once you have everything ready, you can build the tileset:
 ```sh
 # Assuming that you are in the root of the tileset repository
-$ python3 <path-to-compose-py-script>/compose.py gfx/UltimateCataclysm
+$ python3 tools/compose.py gfx/UltimateCataclysm
```

Comment:
we probably should not have a copy of compose in here since the one that actually gets worked on and updatd is in the dda repo

Reply:
hmm I wonder why it was added to the repo in the first place. I remember that you have to have a main cdda repo for this script